HybridCar/src/main/java/Controller/CarBussines.java

```diff
@@ -0,0 +1,58 @@
+package Controller;
+
+import Model.Vehicle;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class CarBussines {
+
+
+    public static List<Vehicle> addVehicle() {
```

Comment:
should not be static. if the name is addVehicle then it should not return a list of vehicles!